lib/kdump_utils.pm

```diff
@@ -274,7 +274,7 @@ sub configure_service {
         }
     }
 
-    prepare_for_kdump($args{test_type});
+    prepare_for_kdump(test_type => $args{test_type});
```

Comment:
i suppose should pass hash like this ? prepare_for_kdump(%args)

Reply:
will try: https://openqa.nue.suse.com/tests/6350633

Reply:
It works and changed the code. thanks